src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_transfer.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "pass_manager.h"
+#include "reorder_inst.h"
+#include "permute_inst.h"
+#include "program_helpers.h"
+
+using namespace cldnn;
+
+void reorder_transfer::run(program& p) {
+    auto itr = p.get_processing_order().begin();
+    while (itr != p.get_processing_order().end()) {
+        auto& node = *itr++;
+
+        if (!node->is_type<reorder>())
+            continue;
+
+        auto& reorder_node = node->as<reorder>();
+
+        bool is_simple_reorder = reorder_node.is_constant() &&
+                                 !reorder_node.is_output() &&
+                                 reorder_node.get_users().size() == 1 &&
+                                 reorder_node.get_dependencies().size() == 1 &&
+                                 !reorder_node.has_mean() &&
+                                 reorder_node.get_primitive()->subtract_per_feature.empty();
+        if (!is_simple_reorder)
+            continue;
+
+        layout input_layout = reorder_node.get_input_layout();
+        layout output_layout = reorder_node.get_output_layout();
+        bool only_precision_changed = input_layout.get_partial_shape() == output_layout.get_partial_shape() &&
+                                      input_layout.format == output_layout.format &&
+                                      input_layout.data_type != output_layout.data_type;
+        if (!only_precision_changed)
+            continue;
```

Comment:
How about add a handle in reorder_node to return whether it is a simple_reorder_with_only_precision_change ? 
I think that condition can be utilized in other locations such as remove redundant reorder too.